drawers/cAllegroDrawer.h

```diff
@@ -29,29 +37,30 @@ class cAllegroDrawer {
 		void stretchSprite(BITMAP *src, BITMAP *dest, int pos_x, int pos_y, int desiredWidth, int desiredHeight);
 
 		void stretchBlit(BITMAP *src, BITMAP *dest, int src_x, int src_y, int width, int height, int pos_x, int pos_y, int desiredWidth, int desiredHeight);
-        void stretchBlitFromGfxData(int index, BITMAP *dest, int src_x, int src_y, int width, int height, int pos_x, int pos_y, int desiredWidth, int desiredHeight);
+    void stretchBlitFromGfxData(int index, BITMAP *dest, int src_x, int src_y, int width, int height, int pos_x, int pos_y, int desiredWidth, int desiredHeight);
 
-        void maskedBlit(BITMAP *src, BITMAP *dest, int src_x, int src_y, int pos_x, int pos_y, int width, int height);
-        void maskedBlitFromGfxData(int index, BITMAP *dest, int src_x, int src_y, int pos_x, int pos_y, int width, int height);
+    void maskedBlit(BITMAP *src, BITMAP *dest, int src_x, int src_y, int pos_x, int pos_y, int width, int height);
+    void maskedBlitFromGfxData(int index, BITMAP *dest, int src_x, int src_y, int pos_x, int pos_y, int width, int height);
 
 		void maskedStretchBlit(BITMAP *src, BITMAP *dest, int src_x, int src_y, int width, int height, int pos_x, int pos_y, int desiredWidth, int desiredHeight);
 		void maskedStretchBlitFromGfxData(int index, BITMAP *dest, int src_x, int src_y, int width, int height, int pos_x, int pos_y, int desiredWidth, int desiredHeight);
 
-        void drawRectangle(BITMAP *dest, int x, int y, int width, int height, int color);
-        void drawRectangle(BITMAP *dest, const cRectangle &pRectangle, int color);
-        void drawRectangleFilled(BITMAP *dest, const cRectangle &pRectangle, int color);
+    void drawRectangle(BITMAP *dest, int x, int y, int width, int height, int color);
+    void drawRectangle(BITMAP *dest, const cRectangle &pRectangle, int color);
+    void drawRectangleFilled(BITMAP *dest, const cRectangle &pRectangle, int color);
+    void drawRectangleTransparentFilled(BITMAP *dest, const cRectangle& rect, int color, int alpha);
 
-        int getColor_BLACK() { return colorBlack; }
+    int getColor_BLACK() { return colorBlack; }
```

Comment:
I do notice we have different tab/spaces/indenting settings. It might be worth setting up some kind of editorconfig and let our IDE's sync with that to make the diffs less noisy.